start_dev.py

```diff
@@ -38,7 +41,8 @@ def check_update(db_type):
     # Get the current date and time
     today = datetime.now()
 
-    config_selection = {'ct':['clinical_update', 'ct_interval'], 'pm':['pubmed_update', 'pm_interval'], 'gnt':['grant_update', 'gnt_interval']}
+    # config_selection = {'ct':['clinical_update', 'ct_interval'], 'pm':['pubmed_update', 'pm_interval'], 'gnt':['grant_update', 'gnt_interval']}
```

Comment:
I think this line needs to be uncommented and the below line commented out, that or use the variable in sysvars.py to make this generalizable